projects/ideabox-groups.md

```diff
@@ -0,0 +1,210 @@
+
+* At the top of the idea list, include a text field labeled "Search".
+* As a user types in the search box, the list of ideas should filter in real time to only display ideas whose title or body include the user's text. The page _should not_ reload.
+* Clearing the search box should restore all the ideas to the list.
+  * There is no need to make persisting changes to the data model to achieve this functionality.
+
+#### Filter by Importance
+
+The application should allow users to filter the idea list based on level of quality.
+
+* Your application should have 3 buttons corresponding to each level of importance (swill, plausible, and genius).
+* When one of the filter buttons is clicked, the TODO list should only display TODOs with the selected importance.
+  * There is no need to make persisting changes to the data model to achieve this functionality.
+
+#### Recent Ideas
+
+The application should only show the ten most recent Ideas on page load.
+
+* The application should contain a button labeled `Show more ideas...`.
+* When a user clicks on the `Show more TODOs...` button, this list should load additional ideas from the past.
+  * There is no need to make persisting changes to the data model to achieve this functionality.
+
+### Extensions:
+
+#### Character Counter
+
+The application is able to count the number of characters inside of the input field in real time.
+
+* As the user types, the character count should increment up.
+* If the user deletes characters, then the character count should decrease.
+
+#### Submit button disabled based on character count
+
+The submit button should be disabled when there is not valid content in both input fields **and** if the input field character count exceeds 120 characters.
+
+#### Layout comps
+
+Desktop layout:
+
+![desktop specs][desktop-base]
+
+Mobile layout:
+
+![mobile specs][mobile-base]
+
+Full mobile layout:
+
+![full mobile layout][mobile-full]
+
+
+Color, font, and icon spec:
+
+![color font and icon specs][button-hover-specs]
+
+#### Link to icon files
+
+You will need the `svg` files for the delete, upvote, and downvote icons. [Here's the link.](https://drive.google.com/open?id=0B_lPnjyMN6-CaHpTQlRUdzNnZ0U)
+
+[desktop-base]: /assets/images/projects/ideabox/ideabox-01.png
+[mobile-base]: /assets/images/projects/ideabox/ideabox-02.png
+[mobile-full]: /assets/images/projects/ideabox/ideabox-03.png
+[button-hover-specs]: /assets/images/projects/ideabox/ideabox-04.png
+
+## Rubric
+
+### Functional Expectations
+
+* [ ]  Novice - Application meets all of the basic functional expectations of create, edit, delete, and those changes persist in `localStorage`
+* [ ]  Advanced Beginner - Application adds 'Changing the quality of an idea' and enables 'Filtering and Searching by Text' as defined in the spec
+* [ ]  Proficient - Applications adds 'Filter by Importance' and 'Recent' Idea as outlined in the spec
```

Comment:
Is 'Recent" Idea specifying it displays the 10 most recent ideas?  And has the functionality to show the rest with a "Show More..." button?